components/chef-ui-library/src/molecules/chef-control-menu/chef-control-menu.tsx

```diff
@@ -125,8 +125,21 @@ export class ChefControlMenu {
     }
   }
 
-  @Listen('keydown.enter')
-  @Listen('keydown.space') handleKeyActivation() {
+  @Listen('keydown') handleKeydown(event: KeyboardEvent) {
+    switch (event.key) {
+      case (' '):
+      case ('Enter'):
+        return this.handleKeyActivation();
+      case ('Escape'):
+        return this.handleEscape();
+      case ('ArrowUp'):
+        return this.handleUp(event);
+      case ('ArrowDown'):
+        return this.handleDown(event);
+    }
+  }
```

Comment:
Stencil decided to deprecate the ability to use the `@Listen()` decorator to bind to specific keyboard event keys: [change docs](https://github.com/ionic-team/stencil/blob/7b7d0725618b2a17181389096ac60de08e1c766f/BREAKING_CHANGES.md#removed-listeneventkey).
```
[ WARN  ] @Listen('eventName.KEY') is deprecated:
Future versions of Stencil will REMOVE the possibility of listening to specific keys,
since the Web platform already provides a built-in alternative using "event.key". Check
out the docs are https://www.w3.org/TR/uievents-key/#named-key-attribute-values
```
I planned to address this deprecation in the final Stencil 1.0 upgrade PR but for some reason these warnings caused the unit specs to [fail in BuildKite](https://buildkite.com/chef-oss/chef-automate-master-verify/builds/3254#96910a59-4688-4dd0-9223-a59ad47e880a/119-326), but not locally 🤔. Easiest fix was to go ahead and fix the deprecated code and [add it to this PR](https://github.com/chef/automate/pull/944/commits/1e140fd15a2bdab5c8a8e6763f918e32ff7a5c88).